recipes/x86_64_level/conda_build_config.yaml

```diff
@@ -0,0 +1,59 @@
+microarchitecture:
+- x86_64
+- x86_64_v2
+- x86_64_v3
+- x86_64_v4
+- nocona
+- core2
+- nehalem
+- westmere
+- sandybridge
+- ivybridge
+- haswell
+- broadwell
+- skylake
+- mic_knl
+- skylake_avx512
+- cannonlake
+- cascadelake
+- icelake
+- k10
+- bulldozer
+- piledriver
+- steamroller
+- excavator
+- zen
+- zen2
+- zen3
```

Comment:
Newest `archspec` release also has `zen4` (level `4`) which is missing here.

Reply:
Thanks. Fixed in latest commit.

Reply:
Are we going to have to patch old packages as new arches come out?

Reply:
Nope. Just build a new package.